src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActiveState-test.js

```diff
@@ -64,7 +64,7 @@ describe('elements/content-sidebar/ActiveState/activity-feed/ActiveState', () =>
     });
 
     test('should correctly render with comments, tasks, versions', () => {
-        const wrapper = shallow(<ActiveState items={items} />).dive();
+        const wrapper = mount(<ActiveState items={items} />);
```

Comment:
We try try to use `shallow` where possible to keep tests isolated and fast. Is changing all the snapshots necessary or can the feature flip logic be tested on its own?

Reply:
It seems like having the feature flag in the component is an implementation detail, when the test only cares about the task component being rendered. I guess we could add a testid to the featureflag wrapping Task/Task identifying it as handling task cards?

Reply:
I'm open to that approach or others that test the new changes; I'd just like to avoid using `mount` here and regenerating all the snapshots.